thirdparty/README.md

```diff
@@ -49,23 +67,17 @@ account-lookup-service:
   account-lookup-service-admin:
     config:
       featureEnableExtendedPartyIdType: true # allows the ALS to support newer THIRD_PARTY_LINK PartyIdType
+...
 ```
 
 And run the following from the `/thirdparty` directory
 
 ```bash
-# install the example dependencies for the thirdparty charts
-# skip these steps if you want to deploy them yourself
-cd chart-auth-svc
-kubectl apply -f ./example_dependencies.yaml
-cd ../chart-consent-oracle
-kubectl apply -f ./example_dependencies.yaml
-
 # install as a standalone chart, alongside Mojaloop
```

Comment:
How does this pick up the changes to the account-lookup charts when it is using the local thirdparty values file (maybe this correct and I will try it ...but not sure)

Reply:
It does not, one would have to "customize" the values to point to the existing ALS service for example.

It would be great if you could do a PR to clarify that config change in the README.